src/core/referencingfeaturelistmodel.h

```diff
@@ -139,7 +140,7 @@ class QFIELD_CORE_EXPORT ReferencingFeatureListModel : public QAbstractItemModel
      * Deletes a feature regarding the referencing layer and the feature id of the selected child
      * \param referencingFeatureId id of the selected child
      */
-    Q_INVOKABLE bool deleteFeature( QgsFeatureId referencingFeatureId );
+    Q_INVOKABLE virtual bool deleteFeature( QgsFeatureId referencingFeatureId );
```

Comment:
How much code are we duplicating in this method? Could we somehow avoid overriding and parametrize the parent method instead?

Reply:
We can by adding `onAfterDeleteFeature` or something like this. Thought about it, but I think in the end we reduce the code duplication with 10 lines, so I do not think it's worth it.

Reply:
We can by adding `onAfterDeleteFeature` or something like this. Thought about it, but I think in the end we reduce the code duplication with 10 lines, so I do not think it's worth it.

Reply:
We can by adding `onAfterDeleteFeature` or something like this. Thought about it, but I think in the end we reduce the code duplication with 10 lines, so I do not think it's worth it.

Reply:
Was that to proof the point of code duplication?

Reply:
Should do the trick https://github.com/opengisch/QField/pull/1976/commits/d552f31b9fc9cef9a3dcab41fa832ed1bfa571f6